packages/cozy-harvest-lib/test/components/TriggerManager.spec.js

```diff
@@ -189,6 +212,52 @@ describe('TriggerManager', () => {
       wrapper.instance().handleSubmit()
       expect(wrapper.props().submitting).toEqual(true)
     })
+
+    it('should call saveAccount without account', () => {
+      const wrapper = shallowWithoutAccount()
+      wrapper.instance().handleSubmit(fixtures.data)
+      expect(saveAccountMock).toHaveBeenCalledWith(
+        fixtures.konnector,
+        fixtures.account
+      )
+    })
+
+    it('should call saveAccount with account', () => {
+      const wrapper = shallowWithAccount()
+      wrapper.instance().handleSubmit(fixtures.data)
+      expect(saveAccountMock).toHaveBeenCalledWith(
+        fixtures.konnector,
+        fixtures.createdAccount
```

Comment:
I find a bit weird that saveAccount is called with first argument a konnector ?

Reply:
`createAccount` is needing a konnector, to know if it needs to create an aggregator account first.

A workaround could be to pass this information as an options or directly into the account.

But you're right, I think it's a right thing to avoid passing a konnector at this point of our code.

I suggest, if you agree, to keep it for a future (little) refactoring.